### DISCUSSION TITLE
Toggle overlay current active window

### DISCUSSION THREAD

**Discussion by mihailigoshin** [kovidgoyal/kitty#7656] [Q&A] [answered]
Hello, 

I can find in the documentation how make a new windows overlay but I can't find any info how to make current active window to overlay toggle.

Can someone help me out?

**Comment by kovidgoyal** [comment, marked as answer]
There isnt any such functionality. overlays are meant for transient
windows showing dialogs/scrollback/other kinds of information. If you
wan tto manage a stack of more long lived windows, use the stack layout.
